js-ext-core/package.json

```diff
@@ -61,6 +63,7 @@
     "@ethersproject/units": "^5.7.0",
     "elliptic": "^6.6.0",
     "ethers": "^6.13.5",
-    "lodash-es": "^4.17.21"
+    "lodash.camelcase": "^4.3.0",
+    "lodash.snakecase": "^4.1.1"
```

Comment:
currently lodash's `snakeCase` and `camelCase` is still used for its complexity. these 2 module is commonjs and not raise error when imported by node moduleResolution (worked with esm moduleResolution)